js/index.js

```diff
@@ -37,6 +37,86 @@ const siteContent = {
   },
 };
 
+lineBreak = document.createElement('br')
```

Comment:
See, that ended up being pretty simple right?  Just don't forget to define it with a `const`!